src/rasenmaeher_api/db/people.py

```diff
@@ -22,9 +23,11 @@
 from ..prodcutapihelpers import post_to_all_products
 
 LOGGER = logging.getLogger(__name__)
+CODE_CHAR_COUNT = 12
+CODE_ALPHABET = string.ascii_uppercase + string.digits
```

Comment:
Remove, these are not used for anything here.

Also should probably have just imported them from enrollments in the first place.

Reply:
Removed, or was removed by cherrypics from firstuserapi_orm_rambo